Week4/prep-exercises/1-wallet/README.md

```diff
@@ -0,0 +1,108 @@
+# Wallet Exercise
```

Comment:
When writing this exercise I had assumed that students would have completed the week4 ex5-wallet exercise on which the current exercise builds. As the homework for week 4 will be dropped I think there is now missing context here. Also, ex5-wallet teaches how to use the Chrome debugger wich students will now miss out on. Perhaps we can move ex5-wallet from the homework repo to week 4 here as a prep-exercise and intro into this exercise?

Reply:
That is actually still the case, the wallet exercise is now in week 3 so they will have all this information. We are not losing anything from the other weeks, it is just that students have already done the JavaScript basics during the application process so week 1 and 2 are now split differently and we can attach this week to the end